README.md

```diff
@@ -63,13 +65,17 @@ In the Arguments section, add five (or more) Arguments Passed On Launch to the b
   ```
   --base-path="/Users/YourName/Desktop/YourProjectFolder"
   ```
-- The path within your project to your asset catalog, without a preceding slash (add multiple times if you have multiple asset catalogs:  
+- The path within your project to your asset catalog, without a preceding slash (add multiple times if you have multiple asset catalogs):  
   ```
   --asset-catalog="Resources/Images.xcassets"
   ```  
   ```
   --asset-catalog="Resources/Media.xcassets"
   ```
+- The prefix for the category method names. If not specified, the default is `ac`. Must be at least 2 characters, and conform to valid method naming rules:
```

Comment:
These are good questions that I hadn't thought of.

I suppose the prefixes should be added to an array, just like the asset names are, and applied in the same order as on the command line. I'm thinking that in the case where more catalogs are specified than prefixes, the last prefix specified will be used for the remaining catalogs, for the case where all catalogs should share the same prefix; on the other hand, more prefixes than catalogs will result in the leftover prefixes being ignored. Do you agree? Or have another ideas?